karma.conf.js

```diff
@@ -0,0 +1,48 @@
+var path = require('path');
+var webpack = require('webpack');
+
+module.exports = function (config) {
+
+  // To specify browsers to test against use BROWSER var and comma separated list:
+  // BROWSER=Firefox,Chrome,Safari,IE npm run test:browser
+  var browsers = process.env.BROWSER ? process.env.BROWSER.split(',') : [ 'Firefox' ];
```

Comment:
Hm, haven't tested it with npm scripts though. Does `npm run test:browser --browsers Chrome,Firefox` do the right thing?